libs/wingsdk/package.json

```diff
@@ -105,6 +106,7 @@
     "esbuild-wasm": "^0.18.20",
     "express": "^4.18.2",
     "ioredis": "^5.3.2",
+    "js-yaml": "^4.1.0",
```

Comment:
We actually already have a dependency on https://www.npmjs.com/package/yaml, could that be used instead?

Reply:
Yeah we definitely want to minimize deps (even though there are quite a lot already).

Second output makes sense as a default, so this should work:

```js
yaml.stringify(data, { aliasDuplicateObjects: false })
```

Reply:
Oh! Finally! Thank you!